safenode/src/bin/kadnode.rs

```diff
@@ -62,18 +82,32 @@ struct Opt {
     /// Defaults to 0.0.0.0, which will bind to all network interfaces.
     #[clap(long, default_value_t = IpAddr::V4(Ipv4Addr::UNSPECIFIED))]
     ip: IpAddr,
+
+    /// Bootstrap node to help us get connected to the network. Can be specified multiple times.
```

Comment:
better have an example to show how the `peers` opt got used
btw. `MultiAddrs` seems a complex format to be used via CLI. maybe we don't need this opt at all, just read from an input file directly? like the previous network_contacts

Reply:
The network contacts file could also be configured via a file. I understand this is a complex format, but I think it's reasonable for the time being. In my personal opinion, I am fine with leaving out the complexity of a config json.